src/components/withPrepareCentralPaneScreen/index.native.tsx

```diff
@@ -0,0 +1,23 @@
+import type {ComponentType, ForwardedRef, RefAttributes} from 'react';
+import React from 'react';
+import getComponentDisplayName from '@libs/getComponentDisplayName';
+import FreezeWrapper from '@libs/Navigation/FreezeWrapper';
+
```

Comment:
We probably need some comments here and in the other file about what preparing means for native and web.